src/av2/evaluation/detection/eval.py

```diff
@@ -181,6 +181,158 @@ def evaluate(
     return dts, gts, metrics
 
 
+def evaluate_hierarchy(
+    dts: pd.DataFrame,
+    gts: pd.DataFrame,
+    cfg: DetectionCfg,
+    n_jobs: int = 8,
+) -> Tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame]:
+    """Evaluate a set of detections against the ground truth annotations.
+
+    Each sweep is processed independently, computing assignment between detections and ground truth annotations.
+
+    Args:
+        dts: (N,14) Table of detections.
+        gts: (M,15) Table of ground truth annotations.
+        cfg: Detection configuration.
+        n_jobs: Number of jobs running concurrently during evaluation.
+
+
+    Returns:
```

Comment:
Since the return type is a tuple of individual dataframes, it might be nice to explain what each of the return values within the tuple represent.